src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { PeopleContext } from '../../peopleContext';
+import { getPersonByName } from '../../utils/getPeopleByName';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug: userId, sex, born, died, fatherName, motherName } = person;
+  const { slug: selectedUserId } = useParams();
+
+  const people = useContext(PeopleContext);
+  const mother = getPersonByName(motherName, people);
+  const father = getPersonByName(fatherName, people);
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': userId === selectedUserId,
+      })}
+    >
```

Comment:
You're using the `classNames` library to conditionally add a class to the `tr` element. This is a good approach, but it could be simplified. Since you're only adding one class, you could use a ternary operator instead. This would make the code cleaner and easier to understand.